### DISCUSSION TITLE
Tutorial MiniMap not synced with world map

### DISCUSSION THREAD

**Discussion by tvleavitt** [triplea-game/triplea#12116] [General]
The post title is awkwardly phrased, but as you can see, the tutorial "minimap" in the upper right-hand corner only covers the top portion of the overall world map, and thus the "selection" box in it doesn't correlate to what is displayed in the primary viewing area. Is this the result of a change in how 2.6 renders this? I imagine this problem isn't limited to the Tutorial map.

![Screenshot 2023-11-23 at 8 54 16 AM](https://github.com/triplea-game/triplea/assets/4445057/0e6fa730-d897-4fca-ae9e-6e700fec089e)

**Comment by DanVanAtta** [comment]
@tvleavitt  let's track this as an 'issue'; I see you created one already.
